app/abilities/api_ability.rb

```diff
@@ -3,12 +3,16 @@
 class ApiAbility
   include CanCan::Ability
 
-  def initialize(user)
+  def initialize(token)
     can :read, [:version, :capability, :permission, :map]
 
     if Settings.status != "database_offline"
+      user = (User.find(token.resource_owner_id) if token)
```

Comment:
I don't think the outer parentheses are needed here? They would be if `user` already had a value and you wanted to make sure it was overwritten but that's not the case here.

Reply:
removed parentheses